src/BuildOnWindows.cmake

```diff
@@ -36,7 +38,27 @@ if(BUILD_SEPARATE_OPS)
     # Decouple with PyTorch cmake definition.
     install(TARGETS ${sycl_lib} DESTINATION "${TORCH_INSTALL_LIB_DIR}")
   endforeach()
+  list(APPEND TORCH_XPU_OPS_LIBRARIES torch_xpu_ops)
+  list(APPEND TORCH_XPU_OPS_LIBRARIES torch_xpu_ops_aten)
 elseif(BUILD_SPLIT_KERNEL_LIB OR __INTEL_LLVM_COMPILER LESS 20250001 OR ICX_DATE LESS 20241211)
+  add_library(
+    torch_xpu_ops
+    STATIC
+    ${ATen_XPU_CPP_SRCS}
+    ${ATen_XPU_MKL_SRCS})
```

Comment:
Is it right here? MKL sources are needed by `ATen_XPU_NATIVE_CPP_SRCS`? Operators involving MKL implementation are in `ATen_XPU_NATIVE_CPP_SRCS`.

Reply:
You're right. `${ATen_XPU_MKL_SRCS}` should not be here.